kratos/geometries/quadrilateral_2d_8.h

```diff
@@ -1539,16 +1509,20 @@ template<class TPointType> class Quadrilateral2D8
     {
```

Comment:
the static must be there, since it is used in the static initialization of the GeometryData.

However I THINK you can use it in the other functions.  instead of reimplementing everything